stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/global-header.scss

```diff
@@ -3,167 +3,153 @@
 @import '../../../../assets/scss/breakpoints';
 
 .global-header {
-  @extend %header;
+  .header{
+    @extend %header;
 
-  @include devicebreak(mediumonlytab) {
-    height: 4.6875rem;
-  }
-
-  @include devicebreak(xlarge) {
-    .fluid {
-      margin: 0 $spacing-70;
-    }
-  }
-
-  .top-left {
     @include devicebreak(mediumonlytab) {
-      display: flex;
-
-      .dropdown-language {
-        display: none;
-      }
-    }
-
-    .dropdown-language {
-      margin-left: $spacing-12;
+      height: 4.6875rem;
     }
-  }
 
-  .mid-nav {
-    display: flex;
-    justify-content: center;
-    position: relative;
     @include devicebreak(xlarge) {
-      height: 7.1875rem;
-    }
-    nav{
-      @include devicebreak(mediumonlytab) {
-        height: 4.6875rem;
+      .fluid {
+        margin: 0 $spacing-70;
       }
     }
-    ul {
-      position: absolute;
-      left: 50%;
-      width: 50%;    
-      top: 50%;
-      transform: translateY(-50%);
-      padding-left: 4.375rem;
-      display: flex;
-      height: 100%;
-      align-items: center;
-      z-index: 9;
-
-      &:first-child{
-        right: 50%;
-        left: inherit;
-        justify-content: right;
-        padding-right: 4.375rem;
-        padding-left: 0;
-      }
 
+    .top-left {
       @include devicebreak(mediumonlytab) {
-        display: none;
+        display: flex;
+
+        .dropdown-language {
+          display: none;
+        }
       }
-      li{
-        margin: 0 2.1875rem;
+
+      .dropdown-language {
+        margin-left: $spacing-12;
       }
     }
 
-    .logo {
+    .mid-nav {
+      display: flex;
+      justify-content: center;
       position: relative;
-      z-index: 1;
-      @extend %img_none;
-
       @include devicebreak(xlarge) {
-        @include transform(translateX(-50%));
-
+        height: 7.1875rem;
+      }
+      nav{
+        @include devicebreak(mediumonlytab) {
+          height: 4.6875rem;
+        }
+      }
+      ul {
+        position: absolute;
         left: 50%;
-        position: fixed;
+        width: 50%;    
+        top: 50%;
+        transform: translateY(-50%);
```

Comment:
@Ashish-srij Change this to `@include transform(translateY(-50%));`

Reply:
@shishirphadke Done.